### ISSUE TITLE
Unplanned tasks not shown

### ISSUE THREAD

**Issue by busera** [702573N/Obsidian-Tasks-Timeline#27]
Hi, 
I am using the default (demo) file, however, my unplanned tasks are not counted (or shown). Is there something specific to be considered?

Bonus question: How could I show the ongoing tasks, e.g. "process" tasks as shown in some of the screenshots in the description?

Cheers
Andre

**Comment by busera** [comment]
UPDATE:
I found the solution to my first question. I need to specify the **inbox file**. However, would it be possible to specify a folder to capture the unplanned tasks? Because for each day I am creating a separate project journal page containing new tasks.

**Comment by 702573N** [comment]
You can also use daily notes to capture unplanned tasks. If your project notes matching the daily notes format (you can define by yourself), all unplanned tasks were listed inside the timeline.

I will take your idea as feature request.

________________________________

**Comment by busera** [comment]
Yes I just figured out the daily notes structure :-). I will use this as a workaround for now :-). Thanks.
